### DISCUSSION TITLE
error: globalThis is not defined

### DISCUSSION THREAD

**Discussion by FirefoxNL** [requarks/wiki#4526] [Help / Questions] [answered]
When updating from Wiki.js version 2.5.201 to 2.5.219 I get the following error when starting the instance afterwards.

![image](https://user-images.githubusercontent.com/74513001/135531994-76d773a7-e684-41c2-aa9a-8adba90b5f22.png)

Tried searching for it but since I'm not particularly at home with Node.js, I can't seem to fix this.
Everything else stays the same, same node version, same Linux version, no package updates.

I'm running Ubuntu 20.04.3 LTS, node 10.19.0 and PostgreSQL version 11.12.

**Comment by FirefoxNL** [comment, marked as answer]
Fixed by updating Node.JS to version 14.18.

For Debian and Ubuntu based distributions:
```
curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -
sudo apt-get install -y nodejs
```

**Reply by ARR4NN** [reply]
```
E: The repository 'https://deb.nodesource.com/node_14.x focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Error executing command, exiting
```

**Reply by FirefoxNL** [reply]
This is only for Debian and Ubuntu based distro's sorry, I added that to my post.
For other distro's you can check this page:
[https://nodejs.org/en/download/package-manager/](https://nodejs.org/en/download/package-manager/)

If you are on a Debian based distro and you still get that message, maybe check out this page:
[https://github.com/nodesource/distributions/blob/master/README.md](https://github.com/nodesource/distributions/blob/master/README.md)

**Reply by ARR4NN** [reply]
I use Ubuntu 20.04

**Reply by ARR4NN** [reply]
```
nodejs is already the newest version (10.19.0~dfsg-3ubuntu1).
0 upgraded, 0 newly installed, 0 to remove and 92 not upgraded.
root@arr4nn:~# node -v
v14.17.3
```

**Reply by FirefoxNL** [reply]
Ah gotcha. 10.19 is still installed alongside then. Try `apt update && apt autoremove`

**Reply by ARR4NN** [reply]
Do you have discord i can chat with you on real quick?

**Reply by FirefoxNL** [reply]
For completion; the issue was fixed by running updates, which updated Ubuntu from 20.04 to 20.04.3.
